packages/liveblocks-core/src/room.ts

```diff
@@ -2248,28 +2237,20 @@ function makeStateMachine<
 
     // Storage
     batch,
-    undo,
-    redo,
-    canUndo,
-    canRedo,
-    pauseHistory,
-    resumeHistory,
+    history: {
```

Comment:
does this not change the public interface, or is this re-exported somewhere?

Reply:
It does not, in fact this structure mimicks our external/public Room API. Internally it was structured differently, probably just for legacy reasons.